README.md

```diff
@@ -62,25 +62,30 @@ For more on the language, more examples & comparisons with SQL, visit
 [prql-lang.org][prql website]. To experiment with PRQL in the browser, check out
 [PRQL Playground][prql playground].
 
-## Current Status - April 2023
+## Current Status - August 2023
 
 PRQL is being actively developed by a growing community. It's ready to use by
 the intrepid, either with our supported integrations, or within your own tools,
 using one of our supported language bindings.
 
-PRQL still has some minor bugs and some missing features, and probably is only
-ready to be rolled out to non-technical teams for fairly simple queries.
+PRQL still has some infrequent minor bugs and some missing features, and
```

Comment:
> infrequent minor bug

Sounds a bit defensive and also not entirely true - some bugs are quite major :D